src/test/java/org/apache/drill_web_test_framework/web_ui/steps/NavigationSteps.java

```diff
@@ -16,23 +16,37 @@
  */
 package org.apache.drill_web_test_framework.web_ui.steps;
 
-import org.apache.drill_web_test_framework.web_ui.pages.BasePage;
 import org.apache.drill_web_test_framework.web_ui.pages.NavigationPage;
-import org.apache.drill_web_test_framework.web_ui.pages.StoragePage;
-import org.apache.drill_web_test_framework.web_ui.pages.ProfilesPage;
+
+import static org.apache.drill_web_test_framework.web_ui.pages.BasePage.getPage;
 
 public final class NavigationSteps extends BaseSteps {
 
   public QuerySteps navigateQuery() {
-    BasePage.getPage(NavigationPage.class).navigateQuery();
-    return BaseSteps.getSteps(QuerySteps.class);
+    getPage(NavigationPage.class).navigateQuery();
```

Comment:
Why not create a method say getNavigationPage()? You wouldn't need to pass class as a parameter.

Reply:
Done